setup.py

```diff
@@ -1,8 +1,163 @@
 import site
 import sys
+import os
 from setuptools import setup
+from setuptools.command.build_py import build_py
+from pkgconfig import pkgconfig
+from pycparser import c_parser, preprocess_file
+from pycparser.c_ast import Decl, Typedef, TypeDecl, IdentifierType, Struct, ArrayDecl
 
 # workaround bug https://github.com/pypa/pip/issues/7953
 site.ENABLE_USER_SITE = "--user" in sys.argv[1:]
 
-setup(use_scm_version=True, cffi_modules=["scripts/libtss2_build.py:ffibuilder"])
+
+class type_generator(build_py):
```

Comment:
Is their a way we can put this class somewhere else and import it into setup.py?

Reply:
I tried that but that was the reason that pip install failed, didn't find a solution for it

Reply:
> I tried that but that was the reason that pip install failed, didn't find a solution for it

That's a good enough reason for me :-p. That's a refinement that can always happen later and isn't a blocker.